core/src/test/scala/io/gearpump/transport/MockTransportSerializer.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gearpump.transport
+
+import java.io.DataOutput
+
+import io.gearpump.transport.MockTransportSerializer.NettyMessage
+import io.gearpump.transport.netty.ITransportMessageSerializer
+import org.jboss.netty.buffer.ChannelBuffer
+
+class MockTransportSerializer extends ITransportMessageSerializer{
```

Comment:
This class is in folder ```test``` because the ```NettySpec``` is under ```core``` while ```StreamingTransportSerializer``` in under ```streaming```. So the test need a mock implementation.